### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by mahesh-panchal**
Change log:
* [x] Remove pipelines dir.
* [x] Update README with profile and usage info.
* [x] Replace AGAT with GAAS in Annotation preprocessing pipeline.
* [x] Add `uppmax` profile.

### CODE REVIEW

File: AnnotationPreprocessing/nextflow.config [hunk 1]
```diff
@@ -10,6 +10,21 @@ manifest {
...
 
+    uppmax {
+        executor {
+            name = 'slurm'
+        }
+        process {
+            scratch = '$SNIC_TMP'
+        }
+        includeConfig "$baseDir/config/compute_resources.config"
+        env.PERL5LIB='${PERL5LIB:-""}:${HOME}/git/NBIS/GAAS/annotation/'
```
Comment (Juke34) [inline AnnotationPreprocessing/nextflow.config]:
```
      includeConfig "$baseDir/config/compute_resources.config"
         env.PERL5LIB='${PERL5LIB:-""}:${HOME}/git/NBIS/GAAS/annotation/'
```
As we are using GAAS from conda It should be removed, isn't it?

Comment (mahesh-panchal) [reply AnnotationPreprocessing/nextflow.config]: Good point, yes.

**Comment by mahesh-panchal** [comment]
I don't think there is any reason to keep these repos with the same folder structure, and that folder is just unnecessary keystrokes. Currently I see this repository as a repository for Nextflow Annotation pipelines. The bpipe repository is simply now an archive. I don't believe anyone is using it anymore. I don't see anyone contributing workflows to this repository outside of the Annotation scope here. They will most likely be separated into other repositories. This is better from an organisational stand point because then one doesn't download everything every time. As a result, I don't think that the number of pipelines will get too large either. 

In terms of the swap to DLS2 and using modules, at the moment the workflows simply don't need a more elaborate folder structure. In the long term it may also be beneficial to use the nf-core/modules rather than recreate our own. The best case is that we make this repo mature enough to become a nf-core annotation pipeline.

**Comment by mahesh-panchal** [comment]
Made some more updates:
* [x] Updated pipeline README's to follow same format. 
* [x] Checked pipeline parameters and removed redundant parameters.
* [x] Included params.config template to copy rather than generate from `grep`.
* [x] Make singularity as suggested software profile rather than conda for reproducibility.

**Comment by LucileSol** [comment]
if I do : 
```
conda activate /active/me/work/conda/gaas-f0acbc7203062da92
```
and then
```
 fasta_statisticsAndPlot.pl
```
it says
fasta_statisticsAndPlot.pl: command not found

there is a problem with the GAAS package, I think!

**Comment by Juke34** [comment]
Must be replaced by `gaas_fasta_statistics.pl`

**Comment by LucileSol** [comment]
my bad, same problem command not found

**Comment by Juke34** [comment]
Must use GAAS-v1.0.1

**Comment by Juke34** [comment]
I checked the code is fine

**Comment by mahesh-panchal** [comment]
I cannot pull the GAAS docker container from biocontainers (conda).
```
singularity pull quay.io/biocontainers/gaas:1.0.1--pl526r35_0
FATAL:   While pulling library image: image quay.io/biocontainers/gaas:1.0.1--pl526r35_0 (amd64) does not exist in the library
```

**Comment by mahesh-panchal** [comment]
> I cannot pull the GAAS docker container from biocontainers (conda).
> 
> ```
> singularity pull quay.io/biocontainers/gaas:1.0.1--pl526r35_0
> FATAL:   While pulling library image: image quay.io/biocontainers/gaas:1.0.1--pl526r35_0 (amd64) does not exist in the library
> ```

Never mind. I forgot the `docker://` prefix.

**Comment by mahesh-panchal** [comment]
OK.
`nextflow run -profile nbis,singularity,test AnnotationPreprocessing` is resulting in an error by `busco`. Something for me to fix.

**Comment by mahesh-panchal** [comment]
Is it ok to merge?

**Pull request merged**